ci-operator/step-registry/aks/provision/aks-provision-commands.sh

```diff
@@ -35,6 +42,7 @@ fi
 echo "Creating AKS cluster"
 eval "${AKE_CREATE_COMMAND[*]}"
 echo "$CLUSTER" > "${SHARED_DIR}/cluster-name"
+echo "$RESOURCEGROUP" > "${SHARED_DIR}/resourcegroup_aks"
```

Comment:
This line should be next to line 20 where it was created to avoid missing it when some error happened in the middle of lines.

Reply:
Yup, updated.